codepropertygraph/src/main/scala/io/shiftleft/codepropertygraph/cpgloading/CpgOverlayLoader.scala

```diff
@@ -46,17 +48,35 @@ private[cpgloading] object CpgOverlayLoader {
 private class CpgOverlayApplier(graph: ScalaGraph) {
 
   private val overlayNodeIdToSrcGraphNode: mutable.HashMap[Long, Vertex] = mutable.HashMap()
+  private var inverseBuilder: DiffGraph.InverseBuilder = _
 
   /**
     * Applies diff to existing (loaded) OdbGraph
     */
   def applyDiff(overlay: CpgOverlay): Unit = {
+    inverseBuilder = makeInverseBuilder(false)
```

Comment:
to understand the meaning one has to also read `makeInverseBuilder`, because it's unclear what that boolean argument stands for. it's only a few lines below, but i'd still prefer something like
a) inline it, i.e. replace this line  with `inverseBuilder = DiffGraph.InverseBuilder.noop`
b) use named argument

Reply:
Yes, this would be more readable, thanks